applications/Unity.GrantManager/modules/Unity.Flex/src/Unity.Flex.Web/Views/Shared/Components/QuestionDefinitionWidget/Default.cshtml

```diff
@@ -1,11 +1,11 @@
-﻿#pragma warning disable S1128
+﻿@* #pragma warning disable S1128 *@
```

Comment:
Going to check if this is rendered

Reply:
Please fix conflicts for this merge by merging dev in and resolving the conflicts